packages/vuetify/src/components/VCombobox/VCombobox.js

```diff
@@ -60,10 +60,10 @@ export default {
         this.delimiters
```

Comment:
```suggestion
         this.delimiters.length
```

It defaults to an array, so the current condition is always true